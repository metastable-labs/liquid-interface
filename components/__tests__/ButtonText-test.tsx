import * as React from 'react';
import renderer from 'react-test-renderer';

import LQDButton from '../button';

it(`button renders correctly`, () => {
  const tree = renderer.create(<LQDButton title="Hello" />).toJSON();
  expect(tree).toMatchSnapshot();
});
