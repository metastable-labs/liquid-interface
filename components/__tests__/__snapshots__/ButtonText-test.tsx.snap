// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button renders correctly 1`] = `
<View
  accessibilityState={
    {
      "busy": undefined,
      "checked": undefined,
      "disabled": undefined,
      "expanded": undefined,
      "selected": undefined,
    }
  }
  accessibilityValue={
    {
      "max": undefined,
      "min": undefined,
      "now": undefined,
      "text": undefined,
    }
  }
  accessible={true}
  collapsable={false}
  focusable={false}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    {
      "opacity": 1,
    }
  }
>
  <View
    style={
      [
        {
          "alignItems": "center",
          "flexDirection": "row",
          "justifyContent": "center",
        },
        {
          "backgroundColor": "#FFF",
          "borderRadius": 50,
          "paddingBottom": 18,
          "paddingTop": 19,
        },
        {
          "borderColor": "#EBEBEB",
          "borderWidth": 1,
          "elevation": 1,
          "shadowColor": "rgba(62, 62, 62, 0.04)",
          "shadowOffset": {
            "height": -2.4,
            "width": 0,
          },
          "shadowOpacity": 1,
          "shadowRadius": 0,
        },
        {
          "width": "100%",
        },
        undefined,
        null,
      ]
    }
  >
    <Text
      style={
        {
          "color": "#1A2001",
          "fontFamily": "QuantaGroteskProSemiBold",
          "fontSize": 18,
          "fontWeight": "600",
          "lineHeight": 16,
        }
      }
    >
      Hello
    </Text>
  </View>
</View>
`;
